src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -203,7 +205,18 @@ public List<RESTEnkelvoudigInformatieobject> listEnkelvoudigInformatieobjecten(f
         if (zoekParameters.toonGekoppeldeZaakDocumenten) {
             final List<RESTEnkelvoudigInformatieobject> list = new ArrayList<>(result);
             list.addAll(listGekoppeldeZaakInformatieObjectenVoorZaak(zaak));
-            return list;
+            result = list;
+        }
+        if (zoekParameters.ophalenVoorBesluitType != null) {
+            final Besluittype besluittype = ztcClientService.readBesluittype(zoekParameters.ophalenVoorBesluitType);
+            final List<UUID> compareList = besluittype.getInformatieobjecttypen().stream()
+                    .map(informatieobjectType -> {
```

Comment:
we hebben een UriUtil om een uuid van een uuid af te halen

Reply:
Opgelost in opvolgende commit